README.md

```diff
@@ -1 +1,44 @@
-# eslint-config-typescript
\ No newline at end of file
+# eslint-config-typescript
+
+> [TypeScript ESLint](https://github.com/typescript-eslint/typescript-eslint) config for Thesis wide projects.
+
+## Installation
+
+`yarn add -D https://github.com/thesis/eslint-config-typescript.git`
+
+## Usage
+
+### Setting up a project
+
+ 1. Install the linter and config - `yarn add -D https://github.com/thesis/eslint-config-typescript.git`
+ 2. In your `.eslintrc` add:
+ ```js
+{
+  "extends": ["eslint-config-typescript"],
```

Comment:
If we rename the package, this can be `["@thesis-co"]` with the `eslint-config` assumed

Reply:
I wasn't aware we can do this, done in 5cefeed